src/test/java/com/courtcanva/ccfranchise/utils/SuburbTestHelper.java

```diff
@@ -13,27 +14,19 @@
 import java.util.Set;
 
 public class SuburbTestHelper {
-    public static SuburbPostDto createSuburbPostDto() {
-        return SuburbPostDto.builder()
-                .sscCode(11344L)
-                .build();
-    }
 
-    public static SuburbGetDto createSuburbGetDto() {
-        return SuburbGetDto.builder()
-                .sscCode(11344L)
-                .suburbName("East Albury")
-                .postcode(2640)
-                .state(AUState.NSW)
-                .build();
+    public static SuburbListAndFilterModePostDto createSuburbListPostDtoWithIncludeMode() {
+        List<SuburbPostDto> suburbs = new ArrayList<>();
+        suburbs.add(new SuburbPostDto(11344L));
+        suburbs.add(new SuburbPostDto(12287L));
```

Comment:
can be simplified to
        `List<SuburbPostDto> suburbs = List.of(new SuburbPostDto(11344L), suburbs.add(new SuburbPostDto(12287L))`